docs/BackupNoSQL.md

```diff
@@ -0,0 +1,153 @@
+
+1. Pause Scalar Product pods using `scalar-admin`.
+
+   Please refer to the [How to use scalar-admin on a Kubernetes environment](./BackupNoSQL.md#how-to-use-scalar-admin-on-a-kubernetes-environment) section in this document for more details on the pause operation.
+
+1. Check the pause completed time.
+
+   You must note the pause completed time to decide the concrete **period** that you can restore data using the PITR feature.
+
+1. Take a backup using the backup feature of each database.
+
+   If you enable the automatic backup and PITR feature, all you have to do is wait for about 10 seconds since the managed databases take backup automatically. This 10 seconds period is the **period** that we can restore data by PITR.
+
+1. Unpause Scalar Product pods using `scalar-admin`.
+
+   Please refer to the [How to use scalar-admin on a Kubernetes environment](./BackupNoSQL.md#how-to-use-scalar-admin-on-a-kubernetes-environment) section in this document for more details on the unpause operation.
+
+1. Check the unpause started time.
+
+   You must note the unpause started time to decide the concrete **period** that you can restore data using the PITR feature.
+
+1. Check the pod status after backup.
+
+   You must check the following four points using the `kubectl get pod` command after you finish the backup operation.
+   * The number of Scalar product pods (You must compare this number with the number before the backup).
+   * The Scalar product pod names in the `NAME` column (You must compare these pod names with the pod names before the backup).
+   * The Scalar product pod status is `Running` in the `STATUS` column.
+   * The restart counts of each pod in the `RESTARTS` column (You must compare these restart counts with the restart counts before the backup).
+
+   **If the two values are different, you must re-try the backup operation all over again.** This is because, if some pods are added or restarted, those pods run with a `unpause` state. The `unpause` state pods cause data inconsistency in the backup data.
+
+1. (DynamoDB only) You need to do additional operations to create a backup if you use the PITR feature of DynamoDB since it restores data with another name table by PITR. You should these additional steps after you complete this creating **period** steps. Please refer to the [Restore database guide](./RestoreDatabase.md#amazon-dynamodb) for more details on these steps.
+
+## Multiple databases backup
+
+If you use more than two databases under the Scalar products using [Multi-storage Transactions](https://github.com/scalar-labs/scalardb/blob/master/docs/multi-storage-transactions.md) or [Two-phase Commit Transactions](https://github.com/scalar-labs/scalardb/blob/master/docs/two-phase-commit-transactions.md), you must pause all Scalar products at the same time and create the same **period** for multiple databases.
```

Comment:
```suggestion
If you use more than two databases under the Scalar products using [Multi-storage Transactions](https://github.com/scalar-labs/scalardb/blob/master/docs/multi-storage-transactions.md) or [Two-phase Commit Transactions](https://github.com/scalar-labs/scalardb/blob/master/docs/two-phase-commit-transactions.md), you must pause all Scalar products together and create a period where no on-going transactions exist in the databases.
```

Reply:
```suggestion
If you are using more than two databases with ScalarDB or ScalarDL that use [Multi-storage Transactions](https://github.com/scalar-labs/scalardb/blob/master/docs/multi-storage-transactions.md) or [Two-phase Commit Transactions](https://github.com/scalar-labs/scalardb/blob/master/docs/two-phase-commit-transactions.md), you must pause all database instances on ScalarDB or ScalarDL together and create the same period where no ongoing transactions exist in the databases.
```

Reply:
As I mentioned in another comment, I think it would be better to describe `databases that are used by ScalarDB or ScalarDL` instead of `databases that use ScalarDB or ScalarDL`.

What do you think?